mitodl/bootcamp-ecommerce
# add copy bootcamp feature

#### What are the relevant tickets?
fixes https://github.com/mitodl/bootcamp-ecommerce/issues/1057
#### What's this PR do?
It allows the creation of a `new Bootcamp Page` without requiring `BootcampRun`.

#### How should this be manually tested?
* Copy a BootcampRunPage through cms.
* You will observe that a new run page has been created and it has no bootcamp_run selected.

#### Screenshots (if appropriate)
Following are the steps of the new use-case:
* Copy an existing Bootcamp Run Page
![image](https://user-images.githubusercontent.com/42243411/96701979-ab0c2500-13aa-11eb-869f-1fa02657c6e1.png)
* Change its Slug and Title according to your need and click `Copy this Page`
![image](https://user-images.githubusercontent.com/42243411/96702196-f6263800-13aa-11eb-8ffd-17e63a0071d1.png)
* You will see a new Bootcamp Run Page entry
![image](https://user-images.githubusercontent.com/42243411/96702410-27066d00-13ab-11eb-95c0-d574e930f482.png)
* Open the new Bootcamp Run Page and select the required Bootcamp run using the dropdown.
![image](https://user-images.githubusercontent.com/42243411/96702678-6df46280-13ab-11eb-840c-d53906030c9d.png)
